scipy/stats/tests/test_distributions.py

```diff
@@ -475,15 +475,16 @@ def test_mean(self):
     #     df = mp.mpf(df)
     #     half_df = 0.5 * df
     #     entropy = mp.log(mp.gamma(half_df)) + 0.5 * \
-    #               (df - mp.log(2) - (df -1) * mp.digamma(half_df))
+    #               (df - mp.log(2) - (df - mp.one) * mp.digamma(half_df))
     #     return float(entropy)
 
     @pytest.mark.parametrize('df, ref',
-                             [(1, 0.7257913526447274),
-                              (500, 1.0720309422146606),
-                              (1e8, 1.0723649412580334)])
+                             [(1e-4, -9989.7316027504),
+                              (1, 0.7257913526447274),
+                              (1e3, 1.0721981095025448),
+                              (1e10, 1.0723649429080335)])
     def test_entropy(self, df, ref):
-        assert_allclose(stats.chi(df).entropy(), ref)
+        assert_allclose(stats.chi(df).entropy(), ref, rtol=1e-12)
```

Comment:
Taken on its own (as a non-expert), this seems pretty encouraging--the test stringency has increased, new cases have been added, and even if I restore the full set of old cases they still all pass on this branch:

```diff
--- a/scipy/stats/tests/test_distributions.py
+++ b/scipy/stats/tests/test_distributions.py
@@ -481,7 +481,9 @@ class TestChi:
     @pytest.mark.parametrize('df, ref',
                              [(1e-4, -9989.7316027504),
                               (1, 0.7257913526447274),
+                              (500, 1.0720309422146606),
                               (1e3, 1.0721981095025448),
+                              (1e8, 1.0723649412580334),
                               (1e10, 1.0723649429080335)])
     def test_entropy(self, df, ref):
         assert_allclose(stats.chi(df).entropy(), ref, rtol=1e-12)
```

I'm guessing that's not necessary because you've improved the "spread" of the cases, but wanted to double check.